### ISSUE TITLE
Duplicate form elements being submitted in 1.0.1

### ISSUE THREAD

**Issue by mattburton** [LeadDyno/intercooler-js#130]
After updating to 1.0.1 I'm seeing the following behavior, where form inputs are being submitted in duplicate. Entering a value of "asdf" into this form:

```
<form id="addCategoryForm" ic-post-to="/categories/create" style="">
    <div class="form-group">
        <input type="text" class="form-control" name="Name" placeholder="Category Name - 'Apparel', 'Drinkware', etc..." required="">
    </div>
    <button type="submit" class="btn btn-default">Save</button>
    &nbsp;
    <a id="cancelAddCategory" href="javascript:;">Cancel</a>
</form>
```

Results in this form being posted to the backend:

ic-request:true
Name:asdfic-request=true
Name:asdf
undefined:undefined
ic-element-id:addCategoryForm
ic-id:2
ic-target-id:addCategoryForm
ic-trigger-id:addCategoryForm
ic-current-url:/
_method:POST

Downgrading to 1.0.0 fixes the issue for me. The only thing funky going on is the form is contained in an HTML fragment that is loaded into the parent page using Intercooler, but I've never seen any issues like this previously using that method. Let me know if there's anything else you need on this - thanks!

**Comment by carsongross** [comment]
Ugh, I’ll look into it.

Sorry about that!  What browser was this in?  I have tried to replicate it locally in the unit tests and I’m not seeing the issue...

Cheers,
Carson

On October 17, 2016 at 12:21:31 PM, Matt Burton (notifications@github.com) wrote:

After updating to 1.0.1 I'm seeing the following behavior, where form inputs are being submitted in duplicate. Entering a value of "asdf" into this form:

<form id="addCategoryForm" ic-post-to="/categories/create" style="">
    <div class="form-group">
        <input type="text" class="form-control" name="Name" placeholder="Category Name - 'Apparel', 'Drinkware', etc..." required="">
    </div>
    <button type="submit" class="btn btn-default">Save</button>
    &nbsp;
    <a id="cancelAddCategory" href="javascript:;">Cancel</a>
</form>
Results in this form being posted to the backend:

ic-request:true
Name:asdfic-request=true
Name:asdf
undefined:undefined
ic-element-id:addCategoryForm
ic-id:2
ic-target-id:addCategoryForm
ic-trigger-id:addCategoryForm
ic-current-url:/
_method:POST

Downgrading to 1.0.0 fixes the issue for me. The only thing funky going on is the form is contained in an HTML fragment that is loaded into the parent page using Intercooler, but I've never seen any issues like this previously using that method. Let me know if there's anything else you need on this - thanks!

**Comment by mattburton** [comment]
Sure thing - this was in Chrome (53) on both PC and Mac. I can try to come up with a better example tomorrow for you - thanks for taking a look.

**Comment by carsongross** [comment]
Hey Matt,

Found the underlying issue and fixed it, and published v1.0.2.

Please trying it out and let me know.  Thanks for the bug report!

**Comment by mattburton** [comment]
@carsongross yep - that did it - thanks for the quick turnaround!
